app/views/foreman_tasks/tasks/show.html.erb

```diff
@@ -1,5 +1,20 @@
 <% stylesheet 'foreman_tasks/tasks' %>
 
+<%= breadcrumbs(
+  items: [
+    {
+      caption: _('Tasks'),
+      url: url_for(foreman_tasks_tasks_path)
+    },
+    {
+      caption: truncate(format_task_input(@task), :length => 50)
+    }
+  ],
+  name_field: 'label',
```

Comment:
@sharvit could you use `humanized.action` instead of `label`?
The camel-case action names in the switcher list are not very readable. It's inconsistent with what we display in the breadcrumbs too.

Reply:
There are two ways we can achieve that:

1. Expand the core logic so we can pass `name_field: 'humanized.action'` and nested export it from the object.

2. Expand the plugin so it will render a new field `title?` at the root level of the json.  :+1: 

Any thoughts? @tstrachota @iNecas

Reply:
I tend to prefer updating the breadcrumbs logic. Similar problem can appear elsewhere too.

Maybe more flexible alternative to parsing a path to the field could be using a mapping function. Something like this:
```js
nameField: record => record.humanized.action
```
But I'd still keep the possibility to use a string for the simple cases.

Reply:
@tstrachota I don't think it's trivial to use a ruby-mapper inside javascript. Or put a js-mapper inside a ruby hash.

I will implement a nested extraction at the core and will update this PR soon.